assets/src/application/services/context/ToasterProvider.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { createContext } from 'react';
+
+import useSystemNotificationsManager from '../toaster/useSystemNotificationsManager';
+import { SystemNotificationsToaster } from '../toaster/types';
+import { useRenderToaster } from '@appServices/toaster';
+
+const ToasterContext = createContext(null);
+
+export type Toasters = {
+	systemNotifications: SystemNotificationsToaster;
+};
+
+const ToasterProvider: React.FC = (props) => {
+	const systemNotifications = useSystemNotificationsManager();
+	const toasters: Toasters = { systemNotifications };
+	useRenderToaster(systemNotifications.getToasts);
```

Comment:
hmm, I am more used to see to API more like this:
```
const renderToast = useRenderToaster(systemNotifications.getToasts);
.
.
.
renderToast();

```

Reply:
oh that's a good idea